x-pack/solutions/security/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/insights/workflow_insights_results.tsx

```diff
@@ -122,7 +122,9 @@ export const WorkflowInsightsResults = ({
                       {insight.message}
                     </EuiText>
                     <EuiText size={'xs'} color={'subdued'}>
-                      {item.entries[0].type === 'match' && item.entries[0].value}
+                      {item.entries[0].type === 'match' &&
```

Comment:
Just to make sure: we add more checks, so all of them have to be met in order to be valid?